src/databricks/labs/ucx/contexts/workspace_cli.py

```diff
@@ -169,3 +166,22 @@ def iam_role_creation(self):
             self.workspace_client,
             self.aws_resource_permissions,
         )
+
+    @cached_property
+    def notebook_loader(self) -> NotebookLoader:
+        return NotebookLoader()
+
+
+class LocalCheckoutContext(WorkspaceContext):
```

Comment:
Why do we need a new context?

Reply:
New properties which are only relevant there